lightning/src/util/events.rs

```diff
@@ -246,6 +287,12 @@ impl Writeable for Event {
 					(2, claim_from_onchain_tx, required),
 				});
 			},
+			&Event::ChannelClosed { ref channel_id, ref err } => {
+				8u8.write(writer)?;
```

Comment:
I thought we were making this odd? :)

Reply:
Oh damn, yep I'll got our serialization numbering!